File: src/main/java/com/github/graqr/threshr/model/redsky/products/VideoCaption.java
package com.github.graqr.threshr.model.redsky.products;

import com.fasterxml.jackson.annotation.JsonProperty;
import io.micronaut.serde.annotation.Serdeable;

@Serdeable
public record VideoCaption(
        @JsonProperty("caption_url") String captionUrl,
        String language) {
}

File: src/main/kotlin/com/graqr/threshr/circleStore/model/product/VideoCaption.kt
package com.graqr.threshr.circleStore.model.product

import com.fasterxml.jackson.annotation.JsonProperty
import io.micronaut.serde.annotation.Serdeable

@Serdeable
data class VideoCaption(
    @param:JsonProperty("caption_url") val captionUrl: String, val language: String
) {}
